users/usersource.go

```diff
@@ -17,26 +19,39 @@ type StaticSource struct {
 	nextUser *ring.Ring
 }
 
+// NewStaticSource creates a StaticSource of all lower-cased and sorted from the specified users.
 func NewStaticSource(users ...string) *StaticSource {
+	for i, u := range users {
+		users[i] = strings.ToLower(u)
+	}
 	sort.Strings(users)
+
 	ss := &StaticSource{nextUser: ring.New(len(users))}
 	for _, u := range users {
-		ss.nextUser.Value = u
+		ss.nextUser.Value = strings.ToLower(u)
```

Comment:
Why do we need to call `ToLower` again here?  Didn't we just set all the users to lowercase above?

Reply:
Good catch - this was remnant of a previous change.